docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -10,10 +10,39 @@ default:: `syn-blackbox-exporter`
 
 The namespace in which to deploy this component.
 
+== `helmValues`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+List of helm values to pass to the helm chart. For a list of supported values see: https://github.com/helm/charts/tree/master/stable/prometheus-blackbox-exporter
 
 == Example
 
 [source,yaml]
 ----
-namespace: example-namespace
+namespace: my-blackbox-exporter
```

Comment:
Is the idea to have one namespace per exporter? I'd suggest considering allowing users to have multiple exporters in a single namespace (see https://github.com/projectsyn/component-nfs-subdir-external-provisioner for how that's achievable)